### PR TITLE
Made some minor grammatical changes

### PR DESCRIPTION
**Pull request by OWelton-Rosie**

### CODE REVIEW

File: edudoc/staff-crash-course/staff_crash_course.md [hunk 1]
```diff
@@ -19,18 +19,18 @@ WCA Staff should be familiar with all the documents related to their work as WCA
...
+More broadly speaking, the preferred  approach is “over-communication” in the WCA, which keeps all relevant persons in the loop at all times. If you think there is a topic which other staff members should see, please add them as a Carbon Copy (CC) to the email. For example, if you escalate an issue about a specific Delegate to any Team/Committee, be sure to include their Senior and/or Regional Delegate in CC.
 
 ## Making changes to recipients {.page-break-before}
 
-When you make changes to the recipients, make this clear at the top of the email. If there is a need to include another team/committee to the thread then begin by writing a “+” along with the name of the team/committee so everyone is aware. When removing someone from a thread they were once a part of you write a “-” followed by their name. For Example:
+When you make any changes to the recipients, please make this clear at the top of the email. If there is a need to include another team/committee to the thread then begin by writing a “+” along with the name of the team/committee so everyone is aware. When removing someone from a thread they were once a part of you write a “-” followed by their name. For Example:
```
Comment (dmint789) [inline edudoc/staff-crash-course/staff_crash_course.md:33]: I'd say "to the list of recipients"

File: edudoc/staff-crash-course/staff_crash_course.md [hunk 2]
```diff
@@ -46,20 +46,20 @@ Lastly, for team/committee internal discussion you can take a shortcut by removi
 
 ## Representation
 
-Every WCA Staff member will also have received login credentials for their own personal *@worldcubeassociation.org* email account hosted by GSuite.\
+Every WCA Staff member will also have received login credentials for their *@worldcubeassociation.org* email account hosted by GSuite.\
```
Comment (dmint789) [inline edudoc/staff-crash-course/staff_crash_course.md:49]: I'd keep the word "personal"

File: edudoc/staff-crash-course/staff_crash_course.md [hunk 3]
```diff
@@ -46,20 +46,20 @@ Lastly, for team/committee internal discussion you can take a shortcut by removi
...
 
 ## Mailing Lists
 
 There are several mailing lists which all have their dedicated audience and purposes. All team and committee addresses can be found on the [contact page of the WCA](wca{contact}), along with a brief description of their scope of work.\
 Other significant mailing lists include:
 
   - [reports@worldcubeassociation.org](mailto:reports@worldcubeassociation.org) - Recipient channel for all Delegate reports, worldwide. You can opt-in on your WCA profile settings page.
-  - [staff@worldcubeassociation.org](mailto:staff@worldcubeassociation.org) - This is the channel where communication related to the entire organization takes place. Only the really heavy stuff like global discussions belong here.
+  - [staff@worldcubeassociation.org](mailto:staff@worldcubeassociation.org) - This is the channel where communications related to the entire organization takes place. Only the really heavy stuff like global discussions belong here.
```
Comment (dmint789) [inline edudoc/staff-crash-course/staff_crash_course.md:62]: "Communications takes place" is incorrect. I'd revert this change.

**Review by dmint789 — CHANGES_REQUESTED** [review]

**Comment by OWelton-Rosie** [comment]
@dmint789 sorry for the delay and poor pull, I have made another one. [https://github.com/thewca/wca-documents/pull/443](https://github.com/thewca/wca-documents/pull/443)
